types/state.go

```diff
@@ -61,3 +63,22 @@ func (s *State) NextHeight() uint64 {
 	}
 	return s.Height() + 1
 }
+
+// LoadConsensusFromAppState load rollapp_params from genesis doc app_state to the dymint state
+func (s *State) LoadConsensusFromAppState(appState json.RawMessage) error {
```

Comment:
imo name can be improved. I suggest `SetConsensusParamsFromGenesis`

I don't think where in genesis (in this case, under the `app_state` is interesting information)

Also I don't think that the `Load` is the interesting part about this fucntion.